templates/demo-store/app/routes/($locale).products.$productHandle.tsx

```diff
@@ -220,10 +211,103 @@ export function ProductForm() {
   return (
     <div className="grid gap-10">
       <div className="grid gap-4">
-        <ProductOptions
+        <VariantSelector
           options={product.options}
```

Comment:
Do we need `options` and `variants` separated out? Shouldn't we just have `product={product}` and we just derive options and variants from that?

Reply:
I like having them separate, so that they can come from separate requests. Only `options` is required. The `variants` could be deferred.